middlewares/authorizationJwt.js

```diff
@@ -20,19 +20,23 @@ passport.use(
       if (user) {
         return done(null, user);
       }
-    } catch (err) {
-      return done(err);
+    } catch (e) {
+      return done(e);
     }
   })
 );
 
 const auth = (req, res, next) => {
-  passport.authenticate("jwt", { session: false }, (error, user) => {
-    if (!user || error)
-      return res.status(401).json({ message: "Unauthorized!" });
+  passport.authenticate("jwt", { session: false }, (e, user) => {
```

Comment:
tutaj problemem jest `e`. `e` to typowo `event`. Skracamy go na FE w `addEventListener` do `e`, bo jest tak czesto uzywany. Ale passport z jwt to troche bardziej wyspecjalizowana rzecz, znacznie rzadziej uzywana. Polecam rozwinac `e` do czegos ludzko-znaczacego :)